handlers/PivotalHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { subDays } from 'date-fns';
+
+export const STORY_STATES = ['unscheduled', 'unstarted', 'started', 'finished', 'delivered', 'rejected', 'accepted'];
+const PIVOTAL_API_URL = 'https://www.pivotaltracker.com/services/v5';
+
+class PivotalHandler {
```

Comment:
I think this a good abstraction, but we dont need this to be a class since its not holding any internal state

Reply:
Sounds good! I think we can discuss more on the right approach of doing this, and maybe refactor it again separately. 

What are you thinking, just export the methods from a file called `PivotalHandler`? I Think the doted notation does help us a lot as a reading convention. But I'm open to different style discussions.

Reply:
I think it's fine as it is as a class full of static methods, since we're no longer requiring an object to be instantiated to use it.

In general I probably would use a functional component, you could also write the syntax for that and have a default export that includes the functions so you can also use dot notation as well as the explode imports if you want.

Reply:
I like the above mentioned approach: gives you the namespace you are looking for and at the same time allows you to export methods without defining a class that won't be instantiated anyways.